src/coreclr/src/ToolBox/superpmi/superpmi-shared/methodcontext.h

```diff
@@ -1458,6 +1477,7 @@ enum mcPackets
     Packet_GetJitFlags                                   = 154, // Added 2/3/2016
     Packet_GetJitTimeLogFilename                         = 67,
     Packet_GetJustMyCodeHandle                           = 68,
+    Packet_GetLikelyClass                                = 181, // Added 9/27/2020
```

Comment:
I guess it should be `182` (and comment `// Highest packet number: 181` should be updated)

Reply:
Thanks. Seems like this should have caused a merge conflict.